### PR TITLE
Improvements to User Guide: Getting Started

### PR DESCRIPTION
**Pull request by tomchadwin**

### CODE REVIEW

File: source/docs/user_manual/introduction/getting_started.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ returns::
 
 This option allows you to create a snapshot in PNG format from the current view.
 This comes in handy when you have many projects and want to generate
-snapshots from your data, or when you need to regularily create snapshots of the
+snapshots from your data, or when you need regularly to create snapshots of the
```
Comment (havatv) [inline source/docs/user_manual/introduction/getting_started.rst]:
```suggestion
snapshots from your data, or when you need to create snapshots of the
```

Comment (havatv) [reply source/docs/user_manual/introduction/getting_started.rst]: Could "regularly" be removed, as suggested above?

Comment (tomchadwin) [reply source/docs/user_manual/introduction/getting_started.rst]: Agreed - thanks.

**Comment by havatv** [comment]
Thanks, @tomchadwin !

**Pull request merged**